docs/roadmap/planning.rst

```diff
@@ -30,12 +30,24 @@ Major decisions affecting the Roadmap are discussed during Release Planning Meet
 Release Planning Meetings are open to the public with access coordinated via the #deis IRC channel on Freenode.
 Notes from past meetings are below, along with links to a recording of the entire meeting on YouTube.
 
-January 2016
-~~~~~~~~~~~~
+January 2016 (Deis 1.x LTS release and Deis v2 Alpha)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The next public release planning meeting for Deis will take place on
-**Thursday, January 7th** at **12pm MDT/1900 GMT**. The `event`_ can be added
-directly to your calendar.
+We're still taking feedback and suggestions on what our users would like to
+see in the 1.x LTS release. Feel free to provide your input on the
+`LTS release issue`_ on GitHub.
+
+Over the last month we've been hard at work on Deis v2 which puts the Deis
+workflow people have come to know and love on top of the `Kubernetes`_ platform.
+Aaron walks through a demo of what we've done so far with the release of Deis v2
```

Comment:
Also can you add a link to the demo outline here? https://gist.github.com/arschles/5b7a75a50938913d3eb1